### PR TITLE
Admixer Bid Adapter: change admixerwl to rtbstack alias

### PR DESCRIPTION
**Pull request by dariaboyko**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter  
- [x] Updated bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

Update admixerwl alias and rename it to rtbstack

### CODE REVIEW

File: modules/admixerBidAdapter.js [hunk 1]
```diff
@@ -23,8 +23,8 @@ export const spec = {
    * Determines whether or not the given bid request is valid.
    */
   isBidRequestValid: function (bid) {
-    return bid.bidder === 'admixerwl'
-      ? !!bid.params.clientId && !!bid.params.endpointId
+    return bid.bidder === 'rtbstack'
+      ? !!bid.params.endpointId
```
Comment (patmmccann) [inline modules/admixerBidAdapter.js:27]: So you have a bunch of active aliases but you only want one of them to pass validity checks,?

File: modules/admixerBidAdapter.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const ALIASES = [
   {code: 'futureads', endpoint: 'https://ads.futureads.io/prebid.1.2.aspx'},
   {code: 'smn', endpoint: 'https://ads.smn.rs/prebid.1.2.aspx'},
   {code: 'admixeradx', endpoint: 'https://inv-nets.admixer.net/adxprebid.1.2.aspx'},
-  {code: 'admixerwl', endpoint: 'https://inv-nets-adxwl.admixer.com/adxwlprebid.aspx'},
```
Comment (patmmccann) [inline modules/admixerBidAdapter.js:16]: This appears to break people

File: modules/admixerBidAdapter.js (hunk 1 above)
Comment (dariaboyko) [reply modules/admixerBidAdapter.js:27]: No, all aliases must have zone which is checked in the right part of operator, only rtbstack requires other params

File: modules/admixerBidAdapter.js (hunk 2 above)
Comment (dariaboyko) [reply modules/admixerBidAdapter.js:16]: do you mean renaming? if so than this alias isnt released yet on our side so it wont impact on customers.

**Review by patmmccann — APPROVED** [review]

**Pull request merged**